spinnaker-monitoring-daemon/spinnaker-monitoring/stackdriver_descriptors.py

```diff
@@ -426,12 +428,16 @@ def __transform_rule_to_descriptors(self, spectator_name, rule):
         component['type'] += '__count'
         component['description'] = 'Number of measurements in {timer}.'.format(
             timer='%s__totalTime' % meter_name)
+        if component.get('displayName'):
```

Comment:
Feels like there's a lot of similar code in `meter_is_timer` and `meter_is_distribution` blocks.